mws/sitesmanagement/views/others.py

```diff
@@ -125,10 +125,13 @@ def service_settings(request, service_id):
                 url=reverse(service_settings, kwargs={'service_id': service.id}))
     }
 
+    service_conf = AnsibleConfiguration.objects.filter(service=service, key='os')
```

Comment:
Why can we not use ``service.ansible_configuration.os`` directly in the template if we're doing an extra DB call anyway?

Reply:
good point